lib/python/picongpu/plugins/png.py

```diff
@@ -0,0 +1,207 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sebastian Starke
+License: GPLv3+
+"""
+
+import os
+import matplotlib.image as mpimg
```

Comment:
I wonder if there is an alternative way to read in the PNG.
Would be in principle cool if the "data readers" would not need `matplotlib` but only the plotters...

anyway, ok for now with me.

Reply:
Using scipy would work without a lot of changes. Converting to a bytearray seems to loose the shape of the image matrix: when feeding that bytearray into matplotlib it complains about invalid dimensions for image data.
So if scipy as a dependency would be okay for you then I will change that

Reply:
I checked the `scipy` library (not the [SciPy ecosystem/software stack](https://www.scipy.org/about.html)) dep list and it looks quite small. yes, let's do this.